.semgrep/adapter/type-bid-assignment.yml

```diff
@@ -0,0 +1,61 @@
+rules:
+  - id: type-bid-assignment-check
+    languages:
+      - go
+    message: >
+      Found incorrect assignment made to `Bid` Recommended to either create copy of `bid` and then assign it as `Bid`  
+
+      ```
+        for _, seatBid := range response.SeatBid {
+          for _, bid := range seatBids.Bid {
+            ...
+            seatBid := bid
```

Comment:
Please rename this copy variable to `bidCopy` maybe as we already have a range variable named `seatBid`